lib/trento/discovery/payloads/cluster/cluster_discovery_payload.ex

```diff
@@ -112,6 +133,24 @@ defmodule Trento.Discovery.Payloads.Cluster.ClusterDiscoveryPayload do
 
   defp do_detect_cluster_type(_), do: ClusterType.unknown()
 
+  defp put_hana_architecture(%{"cluster_type" => type} = attrs)
```

Comment:
I think we could rename this to put_hana_classic architecture, this clause is perfectly clear, but the other just put nothing, even a maybe_put_hana_classic_architecture but it's really a nitpick

Reply:
Deal!
I will rename to `maybe_put_hana_classic_architecture` and merge the PR